tests/phpunit/tests/post/getPages.php

```diff
@@ -66,7 +66,7 @@ public function test_get_pages_cache() {
 		// Force last_changed to increment.
 		clean_post_cache( $pages[0]->ID );
 		$this->assertNotEquals( $time1, $time2 = wp_cache_get( 'last_changed', 'posts' ) );
-
+		get_post( $pages[0]->ID );
```

Comment:
What is the addition of this function call here meant to test, a specific side effect that we're trying to catch here? An inline comment, or better yet, a single new test case for this issue would be better. Not a blocker.